schema/ddl/20240312172615_create_stock.sql

```diff
@@ -0,0 +1,24 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE IF NOT EXISTS stocks (
+    id char(26) NOT NULL PRIMARY KEY,
+    item_id char(26) NOT NULL,
+    store_id char(26) NOT NULL,
+    quantity INT NOT NULL,
+    expiration_date TIMESTAMP,
```

Comment:
[imo] 変数名を揃えたいかな！
あと，これにもindexを貼りたい！

```suggestion
    expiration_at timestamptz,
```